src/sqlancer/questdb/gen/QuestDBAlterIndexGenerator.java

```diff
@@ -29,7 +31,17 @@ public static SQLQueryAdapter getQuery(QuestDBGlobalState globalState) {
         sb.append(" ");
 
         sb.append("ALTER COLUMN ");
-        String columnName = table.getFreeColumnName();
+
+        // We should always choose column with SYMBOL type
+        QuestDBColumn columnWithSymbolType = table.getRandomColumnWithFilter("SYMBOL");
+
+        if (columnWithSymbolType == null) {
```

Comment:
We could handle this case directly in the method (see my other comment in that method).